contk/metric/metric.py

```diff
@@ -195,7 +197,7 @@ def close(self):
 
 			* **bleu**: bleu value.
 		'''
-		return corpus_bleu(self.refs, self.hyps, smoothing_function=SmoothingFunction.method7)
+		return {"bleu": corpus_bleu(self.refs, self.hyps, smoothing_function=SmoothingFunction.method7)}
```

Comment:
Have you test this line?
I think only ``SmoothingFunction().method7`` works

Reply:
Yes, `SmoothingFunction().method7` always works when I am testing.  But I will check `corpus_bleu` again.